node-notes-cli/app.js

```diff
@@ -0,0 +1,43 @@
+const fs = require('fs');
+const data = require('./data.json');
+
+/* User can read their notes */
+for (const key in data.notes) {
+  if (process.argv[2] === 'read') {
+    console.log(`${key}: ${data.notes[key]}`);
+  }
+}
```

Comment:
This works, but you should do the `if` with the `for` loop inside. It makes more logical sense.

```js
if (process.argv[2] === 'read') {
  for (const key in data.notes) {
    console.log(`${key}: ${data.notes[key]}`);
  }
}
```